packages/vega-view/src/View.js

```diff
@@ -385,5 +389,8 @@ inherits(View, Dataflow, {
 
   // -- SAVE / RESTORE STATE ----
   getState,
-  setState
+  setState,
+
+  // RESIZE HANDLER
```

Comment:
Is that up to date?

Reply:
Could you clarify the question? The only change is adding a new method via the helper.